app/src/main/java/com/simple/demotest/Main2Activity.java

```diff
@@ -0,0 +1,13 @@
+package com.simple.demotest;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/demotest/Main2Activity.java L4

Wrong order for android.os.Bundle import. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*